crypto/keys/multisig/multisig.go

```diff
@@ -15,6 +15,8 @@ var _ multisigtypes.PubKey = &LegacyAminoPubKey{}
 var _ types.UnpackInterfacesMessage = &LegacyAminoPubKey{}
 
 // NewLegacyAminoPubKey returns a new LegacyAminoPubKey.
+// Multisig can be constructed with multiple same keys - it will increase the power of
+// the owner of that key (he will still need to add multiple signatures in the right order).
```

Comment:
We shouldn't document this behavior.

Reply:
Why?

Reply:
I think it's useful to document the expected behavior. Otherwise we end up with issues and discussions like this.